File: Jordy_Files/Ex_5/plot.c
void plot_data() {
    TFile* file = new TFile("data.root", "READ");
    TTree* tree = (TTree*)file->Get("TTree");

    TH1D* hist = new TH1D("hist", "Histogram of Normally Distributed Random Numbers", 50, -5, 5);
    
    tree->Draw("x >> hist");

    hist->SetLineColor(kBlack);   // Set the histogram line color to black
    hist->SetLineWidth(3);        // Set the line width to make it thicker
    hist->SetFillColor(kYellow);  // Set the fill color to yellow
    gStyle->SetOptStat(0);        // Disable the statistics box

    // Customize the canvas background
    gPad->SetFillColor(kWhite);  // Set the background color of the canvas to white

    hist->GetXaxis()->SetTitle("x values");
    hist->GetYaxis()->SetTitle("Frequency");

    hist->Draw("HIST");

    hist->Fit("gaus");

    file->Close();
}

File: Jordy_Files/Ex_6/plot.py
import ROOT

file = ROOT.TFile("data.root", "READ")
tree = file.Get("TTree")

hist = ROOT.TH1D("hist", "Histogram of Normally Distributed Random Numbers", 50, -5, 5)

tree.Draw("x >> hist")

# Customize the histogram
hist.SetLineColor(ROOT.kBlack)  
hist.SetLineWidth(3)            
hist.SetFillColor(ROOT.kYellow)

# Customize background
ROOT.gStyle.SetOptStat(0) 
ROOT.gPad.SetFillColor(ROOT.kWhite) 

hist.GetXaxis().SetTitle("x values")
hist.GetYaxis().SetTitle("Frequency")

hist.Fit("gaus")

canvas = ROOT.TCanvas("canvas", "Histogram Canvas", 800, 600)
hist.Draw("HIST")
canvas.SaveAs("histogram.png")

file.Close()
print("Histogram plotted and saved as histogram.png")
